_data/navbars/getting-started.yml

```diff
@@ -119,6 +119,11 @@ section:
       path: /getting-started/kubernetes/hardway/istio-integration
   - title: System requirements
     path: /getting-started/kubernetes/requirements
+  - title: Operator customizations
```

Comment:
Wonder if this belongs in "Operations" rather than "Getting started"? This is more of an administrative / operations type thing than a "try Calico out for the first time" type thing. WDYT? 

I also wonder if we can come up with a more descriptive name than "customizations" - maybe just "Calico Operator" and put all of the articles relating to it under that heading?

Reply:
@bmckercher123 WDYT?

Reply:
Given the different architectures between EE/OS, I propose:

**EE - Leave in /Install**
+ Install on clusters
+ Install on non-cluster hosts
+ Install image options
   - Deploy images by digest
   - Deploy image from private registry
+ Upgrade
- Install a patch release
  
OS - Move to /Operations
+ Upgrade
- Deploy images by digest

Reply:
After chatting with @bmckercher123 I'll move this new doc to /Operations/Image options/Deploy images by digest